web3/_utils/method_formatters.py

```diff
@@ -315,6 +315,25 @@ def apply_list_to_array_formatter(formatter: Any) -> Callable[..., Any]:
 )
 
 
+call_without_override: Callable[
+    [Tuple[Dict[str, Any], Union[str, int]]],
+    Tuple[Dict[str, Any], int]
+]
+call_without_override = apply_formatters_to_sequence([
+    transaction_param_formatter,
+    to_hex_if_integer,
+])
+call_with_override: Callable[
+    [Tuple[Dict[str, Any], Union[str, int], Dict[str, Any]]],
```

Comment:
nitpick: could these types use `TxParams`, `BlockIdentifier`, and `CallOverrideParam`?